### ISSUE TITLE
Color difference between preview and render

### ISSUE THREAD

**Issue by m1gu3l** [remotion-dev/remotion#3850]
There's very noticeable difference in how Video and OffthreadVideo handles colors.

From a color, f.e. `#00BDF2`, let's create a JPEG, a PNG and a video.
![blue.jpeg](https://github.com/remotion-dev/remotion/assets/140639/155af666-b4ae-4e11-8e5a-599047438996)

![blue.png](https://github.com/remotion-dev/remotion/assets/140639/871227b3-0365-402c-95f5-a27724a327c0)

```ffmpeg -f lavfi -i color=c=#00BDF2:s=1920x1080:r=30 -vf "format=yuv420p, colorspace=iall=bt601-6-525:all=bt709:fast=1" -t 10 -c:v libx264 -profile:v main output.mp4```

https://github.com/remotion-dev/remotion/assets/140639/24416162-d418-455c-9011-b65ceaed4ea9

Having these assets, let's create Remotion video.

```
<Composition
  id="BlueVideo"
  width={640}
  height={480}
  fps={25}
  durationInFrames={120}
  component={() => (
    <AbsoluteFill>
      <Sequence from={0} durationInFrames={30}>
        <Img src="http://localhost:8080/assets/blue.jpg" />
        <div
          style={{
            position: 'absolute',
            backgroundColor: '#00BDF2',
            padding: 20,
          }}
        >
          JPEG
        </div>
      </Sequence>
      <Sequence from={30} durationInFrames={30}>
        <Img src="http://localhost:8080/assets/blue.png" />
        <div
          style={{
            position: 'absolute',
            backgroundColor: '#00BDF2',
            padding: 20,
          }}
        >
          PNG
        </div>
      </Sequence>
      <Sequence from={60} durationInFrames={30}>
        <Video src="http://localhost:8080/assets/output.mp4" />
        <div
          style={{
            position: 'absolute',
            backgroundColor: '#00BDF2',
            padding: 20,
          }}
        >
          VIDEO
        </div>
      </Sequence>
      <Sequence from={90} durationInFrames={30}>
        <OffthreadVideo src="http://localhost:8080/assets/output.mp4" />
        <div
          style={{
            position: 'absolute',
            backgroundColor: '#00BDF2',
            padding: 20,
          }}
        >
          OFFTHREAD VIDEO
        </div>
      </Sequence>
    </AbsoluteFill>
  )}
/>
```

While previewing in Remotion Studio everything looks fine; after rendering though OffthreadVideo produces invalid color.

https://github.com/remotion-dev/remotion/assets/140639/0234986a-8034-495f-8e6b-32c0168fe668

Tested on current Remotion (4.0.153) and FFmpeg 6.0-6ubuntu1

**Comment by JonnyBurger** [comment]
You can also render in bt709 in Remotion with a flag: https://www.remotion.dev/docs/cli/render#--color-space

We'll make this the default in Remotion 5.0!

**Comment by m1gu3l** [comment]
The final (broken) video **was rendered in BT709**

**Comment by JonnyBurger** [comment]
Alright, thanks for filing!

I will take a look.

**Comment by JonnyBurger** [comment]
Thanks!

I managed to improve the output:

New:
![new](https://github.com/remotion-dev/remotion/assets/1629785/c1d28539-163f-4239-a415-3f63717c5219)

Old:

![old](https://github.com/remotion-dev/remotion/assets/1629785/77e40ab2-8c03-4b9d-9a6e-5595b48d98ac)

When adding a video, Remotion extracts the frames using Rust and returns images to the browser.  
This input video is bt709, so when we extract frames, it is approximately sRGB format, but we returned .bmp.

We get better results when we return .png's instead which we can tag as sRGB.
Chrome interprets this better.

When doing a conversion, there is always a bit of color loss, but it is way less now.

**Comment by m1gu3l** [comment]
That's really close, looking good!

Thanks for quick fix :grinning:

**Comment by JonnyBurger** [comment]
Unfortunately this regressed and was the wrong way to go about it.
It only worked for this shade of blue.

This change will be reverted in 4.0.155.

I'm making another attempt which seems more promising.

**Comment by JonnyBurger** [comment]
Will try to fix this with #3857

**Comment by m1gu3l** [comment]
Is there a way I could support you?
